config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
Not totally sure but this method of initializing OKComputer may be depreciated (see https://github.com/sul-dlss/preservation_catalog/issues/1371)

Reply:
It it still the documented way for OKComputer: https://github.com/sportngin/okcomputer. Not sure of how else to do it.

Reply:
How about wrapping the code in `to_prepare` per https://stackoverflow.com/a/59971682

Reply:
@jermnelson that's only an issue if you call `OkComputer::OkComputerController`, which @justinlittman is not doing here.  

@mjgiarlo that isn't necessary since we're not defining any constants here.